### PR TITLE
control if recaptcha is not checked

### PR DESCRIPTION
**Pull request by javikalsan**
control if recaptcha is not checked

### CODE REVIEW

File: src/Controller/Component/RecaptchaComponent.php [hunk 1]
```diff
@@ -74,8 +74,12 @@ public function startup(Event $event)
         $controller->helpers['Recaptcha.Recaptcha']['type'] = $type;
 
         if ($controller->request->is(['post', 'put'])) {
-            //debug($controller->request->data);
-            // Was there a reCAPTCHA response?
+        
+            // if Recaptcha is not checked
+            if (empty($controller->request->data["g-recaptcha-response"])) {
```
Comment (cake17) [inline src/Controller/Component/RecaptchaComponent.php:79]: Maybe we can add an isset() ?

Comment (cake17) [inline src/Controller/Component/RecaptchaComponent.php:79]: if another form with no recaptcha is submitted in the same page, I don't want it to be redirected

Comment (javikalsan) [inline src/Controller/Component/RecaptchaComponent.php:79]: Yeah and isset is a good idea also.

Comment (javikalsan) [inline src/Controller/Component/RecaptchaComponent.php:79]: Another casuistry: If we have 2 forms with 2 recaptcha?

Comment (cake17) [inline src/Controller/Component/RecaptchaComponent.php:79]: Yes that's something that should be implemented.

I will open a new issue for multiple recaptcha. The recaptcha doc explains how to implement multiple recaptcha [here](https://developers.google.com/recaptcha/docs/display).

**Comment by cake17** [comment]
Thanks to @javikalsan  !! :+1:

**Pull request merged**